README.md

```diff
@@ -1,34 +1,84 @@
-# Sourcegraph TypeScript toolchain 
+# srclib-typescript
+**srclib-typescript** is a [srclib](https://sourcegraph.com/sourcegraph/srclib)
+toolchain that performs TypeScript code analysis: type inference,
+documentation generation, jump-to-definition, dependency resolution, etc.
+It enables this functionality in any client application whose code analysis is
+powered by srclib, including [Sourcegraph](https://sourcegraph.com).
+## Installation
+This toolchain is not a standalone program; it provides additional functionality
+to applications that use [srclib](https://srclib.org).
+First,
+[install the `src` program (see srclib installation instructions)](https://sourcegraph.com/sourcegraph/srclib).
+Then run:
+```
+git clone https://github.com/sourcegraph/srclib-typescript.git
+cd srclib-typescript
+src toolchain add sourcegraph.com/sourcegraph/srclib-typescript
```

Comment:
There is no `srclib toolchain add` anymore, please propose a pull request that will support `srclib toolchain install typescript`